jobs/tests/test_models.py

```diff
@@ -127,3 +128,13 @@ def test_get_previous_approved(self):
     def test_region_optional(self):
         job = self.create_job(region=None)
 
+    def test_email_on_submit(self):
+        job = self.create_job()
+        mail.outbox = []
```

Comment:
Actually that line is necessary. The listener is looking for a `Job` saved with its status equal to `STATUS_REVIEW`, and that's the *default* value of the field. So the factory calls caused by `create_job()` will pollute the outbox, and it's necessary to clear it and then do a manual save and test.